app/assets/stylesheets/common/whcm.scss

```diff
@@ -0,0 +1,33 @@
+// Overrides for Windows High Contrast Mode
+
+/*
+  Some @media (forced-colors: active) {} styles
+  have also been used inside mixins in:
+  - common/components/buttons.scss
+
+  See file above for overrides.
+*/
+
+// Select Kit
+.select-kit {
+  .select-kit-row {
+    &:hover,
+    &:focus {
+      @media (forced-colors: active) {
+        outline: 2px auto transparent;
+      }
+    }
+  }
+}
+
+// Search
+.search-container .search-filters {
+  details.advanced-filters,
+  details.search-advanced-additional-options {
+    > summary {
+      &:focus-visible {
+        outline: 2px solid transparent;
+      }
+    }
+  }
+}
```

Comment:
Since this applies to all browsers it should be moved out of whcm scss file.

Reply:
Good, point, I've moved it back. Thanks :)